.github/workflows/firefly-ci.yml

```
@@ -0,0 +1,27 @@
+name: Firefly CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - 'develop'
+
+jobs:
+  linting:
```

```suggestion
  js:
```

We might want to add SwiftLint and rustfmt later